lib/Target/Mips/MipsSEInstrInfo.cpp

```diff
@@ -225,7 +228,7 @@ storeRegToStack(MachineBasicBlock &MBB, MachineBasicBlock::iterator I,
       llvm_unreachable("Unexpected register type for CHERI!");
     }
     BuildMI(MBB, I, DL, get(Opc)).addReg(SrcReg, getKillRegState(isKill))
-      .addFrameIndex(FI).addImm(0).addMemOperand(MMO)
+      .addFrameIndex(FI).addImm(Offset).addMemOperand(MMO)
```

Comment:
(Assuming you're referring to all the s/0/Offset/ changes, instead of just some subset thereof)